joomla/joomla-cms
# [4.0] Fix display image contain space in filename

Pull Request for Issue #35940.

### Summary of Changes
Currently, if an image contain space in filename is selected in a media form field, it won't be rendered properly. The reason is because media manager encode the filename (replace space character with %20), and that file could not be found by php `is_file` function. This PR attempts to solve it by replacing %20 back to space before `is_file` check.


### Testing Instructions
1. Follow https://github.com/joomla/joomla-cms/issues/35940 , confirm the issue.
2. Apply patch, confirm that the issue is fixed.
